src/Modifiers/CoreModifiers.php

```diff
@@ -2909,6 +2909,10 @@ public function embedUrl($url)
             }
         }
 
+        if (Str::contains($url, 'youtube.com/shorts/')) {
+            $url = str_replace('youtube.com/shorts/', 'youtube.com/embed/', $url);
```

Comment:
Wir sollten den selben Approach verwenden, wie in line 2917 und mit youtube nocookie ersetzen